src/io/flutter/logging/FlutterConsoleLogManager.java

```diff
@@ -322,6 +331,11 @@ private void printDiagnosticsNodeProperty(ConsoleView console, String indent, Di
       if (StringUtil.equals("ErrorSpacer", property.getType())) {
         return;
       }
+
+      if (StringUtil.equals("DevToolsDeepLinkProperty", property.getType()) && FlutterSettings.getInstance().isEnableEmbeddedBrowsers()) {
```

Comment:
when we see a `DevToolsDeepLinkProperty` does this early return guarantee that we don't show the deep link url in the intellij console?

Reply:
Right, the early return skips printing in the console. I could add a short description like "Click on notification to inspect widget" or something like that.